ev3/legoev3_motor.c

```diff
@@ -57,12 +57,11 @@
 #include "../motors/ev3_motor.h"
 
 #define TACHO_MOTOR_POLL_NS	2000000	/* 2 msec */
+#define TACHO_MOTOR_POLL_MS	2000	/* 2 msec */
 
 #define TACHO_SAMPLES		128
 
-#define MAX_PWM_CNT		10000
-#define MAX_SPEED		100
-#define MAX_POWER		100
+#define MAX_DUTY_CYCLE		100
```

Comment:
Probably doesn't make sense to define this here, but rather in the dc-motor class since that is what consumes the duty cycle value.